app/views/moderations/_mod_sidebar_left.html.erb

```diff
@@ -0,0 +1,51 @@
+<div class="crayons-layout__sidebar-left">
+  <p class="fw-heavy fs-3xl" style="margin-bottom: 29px;">Mod Center</p>
+  <div id="mod-center-inbox-toggle" style="display: flex; justify-content: space-between; padding: 8px;">
+    <span class="fw-medium">Inbox</span>
+    <span>*</span>
+  </div>
+  <div id="mod-center-inbox" class="hidden">
+    <% if current_user.moderator_for_tags.any? %>
+      <% current_user.moderator_for_tags.each do |tag| %>
+        <a
+          href="#"
+          class="crayons-link crayons-link--current crayons-link--block"
+          style="display: flex; justify-content: space-between; padding-left: 24px; padding-right: 12px;">
+          # <%= tag %>
+          <span id="<%= tag %>-indicator" class="crayons-indicator">12</span>
+        </a>
+      <% end %>
+    <% end %>
+    <a
+      href="#"
+      class="crayons-link crayons-link--block"
+      style="padding-left: 24px;">
+      All topics
+    </a>
+  </div>
+  <div id="mod-center-other-options">
+    <a href="#" class="crayons-link crayons-link--block">
+      Connect Chat
+    </a>
+    <a href="#" class="crayons-link crayons-link--block">
+      Settings
+    </a>
+    <a href="#" class="crayons-link crayons-link--block">
```

Comment:
This can link to `<%= URL.url %>/community-moderation`

Reply:
yes! thanks 👍